lib/data/rates/energy_grants_calculator.yml

```diff
@@ -0,0 +1,3 @@
+- start_date: 2015-04-06
+  end_date: 3000-01-01
```

Comment:
Is there a test that documents that? I don't remember seeing one. I based my assumptions on the behaviour in the rates query related tests.

I suppose I could add a test for the end date before changing the code here.